paper/paper.md

```diff
@@ -155,16 +156,15 @@ distributions. In particular, it implements a broader form of elastic net regula
 ## Pyglmnet is an extensible pure Python implementation
 
 Pyglmnet implements the algorithm described in [Friedman, J., Hastie, T., & Tibshirani, R. (2010)](https://web.stanford.edu/~hastie/Papers/ESLII.pdf) and the accompanying popular R package [glmnet].
-As opposed to [python-glmnet] or [glmnet_python], which are wrappers around this package, pyglmnet is written in pure Python for Python 3.5+. Therefore it is easier to extend and more compatible with the existing data science ecosystem.
+As opposed to [python-glmnet] or [glmnet_python], which are wrappers around this R package, pyglmnet is written in pure Python for Python 3.5+. Therefore, it is easier to extend and more compatible with the existing data science ecosystem.
 
 ## Pyglmnet is unit-tested and documented with examples
 
-Pyglmnet has already been used in published work
-[@bertran2018active; @rybakken2019decoding; @hofling2019probing; @benjamin2017modern]. It contains unit tests and includes [documentation] in the form of tutorials, docstrings and examples that are run through continuous integration.
+Pyglmnet has already been used in published work [@bertran2018active; @rybakken2019decoding; @hofling2019probing; @benjamin2017modern]. It contains unit tests and includes [documentation] in the form of tutorials, docstrings and examples that are run through continuous integration.
 
 # Example Usage
 
-Here we apply pyglmnet to a real-world example. The Community and Crime dataset, one of 400+ datasets curated by the UC Irvine Machine Learning Repository [@Dua:2019] provides a highly curated set of 128 demographic attributes of US counties that may be used to predict incidence of violent crime. The target variable (violent crime per capita) is normalized to lie in $[0, 1]$. Below, we demonstrate the usage of a binomial-distributed GLM with elastic net regularization.
+Here, we apply pyglmnet to predict incidence of violent crime from the Community and Crime dataset, one of 400+ datasets curated by the UC Irvine Machine Learning Repository [@Dua:2019] provides a highly curated set of 128 demographic attributes of US counties. The target variable (violent crime per capita) is normalized to the range of $[0, 1]$. Below, we demonstrate the usage of a pyglmnet's binomial-distributed GLM with elastic net regularization.
```

Comment:
The first sentence reads weird. We're missing a "which"?

Reply:
yes, it should be "which provides". Can you do a quick edit on that?

Reply:
Sure, I will to do that. 

Btw, note for the future -- it is not a good idea to make PRs from master. Maybe we should add it to our contributing guide. Because as the repository owner, if I push to your master then your master on origin is not a simple fast forward merge from upstream master. So if you accidentally pull from origin instead of upstream, you may have a bad master branch locally. Also, if I force push to this branch, it can close the pull request (recent experience of a lab colleague).

Reply:
@jasmainak nevermind, I did a fix according to your comment!

Reply:
lol okay, we don't need to bother about the story above then :)

Reply:
@jasmainak Yeah, totally agree, I shouldn't make PR from the master branch! I will make PR from my branches for the future PRs.